requirements/test_requirements.txt

```diff
@@ -8,7 +8,7 @@ pytest-cov
 pytest-xdist
 tensorflow
 pytest-timeout
-matplotlib
+matplotlib<3.5.0
```

Comment:
How about `matplotlib!=3.5.*` for now?

Reply:
`matplotlib==3.6.x` will have even more deprecated methods :D. It's about `plotly` that needs an old version so I don't think it will be proper. + This requirements are only for testing purposes.